### PR TITLE
Yyosifov import gestures

### PR DESCRIPTION
**Pull request by yyosifov**
Added "import gestures" in all samples - if this doesn't sound reasonable, it might be a good idea to mention in the beginning that import gestures should be present, for all samples.

Added a short description of the direction value regarding which direction a swipe has been done.

### CODE REVIEW

**Comment by tjvantoll** [comment]
Love adding this, but just fyi the syntax is off.

All JavaScript examples should use the syntax `var gestures = require("ui/gestures");`.
All TypeScript examples should use `import { GestureTypes } from "ui/gestures";`, and then change `gestures.GestureTypes` to just `GestureTypes`.

**Comment by yyosifov** [comment]
@tjvantoll Sure, sorry - fixed it.

File: ui/gestures.md [hunk 1]
```diff
@@ -41,16 +41,18 @@ The next sections introduce you to all the gestures recognized by NativeScript:
...
 ``` JavaScript
+var gestures = require("ui/gestures");
 var labelModule = require("ui/label");
 var label = new labelModule.Label();
 label.on(gestures.GestureTypes.tap, function (args) {
     console.log("Tap");
 });
 ```
 ``` TypeScript
+import { GestureTypes } from "ui/gestures";
```
Comment (tjvantoll) [inline ui/gestures.md:52]: `import { GestureTypes, GestureEventData } from "ui/gestures";`

File: ui/gestures.md [hunk 2]
```diff
@@ -41,16 +41,18 @@ The next sections introduce you to all the gestures recognized by NativeScript:
...
 import labelModule = require("ui/label");
 var label = new labelModule.Label();
-label.on(gestures.GestureTypes.tap, function (args: gestures.GestureEventData) {
+label.on(GestureTypes.tap, function (args: gestures.GestureEventData) {
```
Comment (tjvantoll) [inline ui/gestures.md:55]: `gestures.GestureEventData` --> `GestureEventData`

Comment (tjvantoll) [inline ui/gestures.md:55]: Same general comment applies to most of the other examples in this PR, although sometimes the name of the event data object changes.

**Comment by tjvantoll** [comment]
Hey @yyosifov,

Sorry, but I noticed one that the event data objects also need to be simplified in the TypeScript code. It seems there are a number of things wrong with these examples on the docs right now 😦

**Comment by tjvantoll** [comment]
I’m going to go ahead and merge this and tidy up the last few things. Thanks again @yyosifov!

**Comment by tjvantoll** [comment]
Merged with b1a46c215f890f3f43e50f4a40755a2a4f47f7b9.

**Pull request closed without merging**